compiler/luci/lang/include/luci/IR/Module.h

```diff
@@ -18,6 +18,7 @@
 #define __LUCI_MODULE_H__
 
 #include <loco/IR/Graph.h>
+#include "CircleDataFormat.h"
```

Comment:
I think this should be above `<loco/IR/Graph.h>`

Reply:
@seanshpark I made CircleDataFormat in luci, because data format only exists in circle schema. Also, I don't want to change loco graph (we've talked about it).

Reply:
I'm talking about the header sequence
```cxx
#include "CircleDataFormat.h"

#include <loco/IR/Graph.h>
```
so that `CircleDataFormat.h` file itself has no problem(self complete) with other headers, as you stated `I made CircleDataFormat in luci` so we include _luci_ first then base _loco_ order.

Reply:
Sorry for the late reply. The comment is reasonable. But I'd like to close this PR for now as our schema does not support different data formats in one subgraph.